optimum/habana/transformers/models/qwen2/modeling_qwen2.py

```diff
@@ -325,33 +345,22 @@ def pre_attn_forward(
         else:
             past_key_value = None
 
+        flash_attention_fast_softmax = True  # TODO pass this along
```

Comment:
can you change as pass?

Reply:
there are other models with the same todo. i'll create a PR thsat fixes this todo for all of them in one go